Make ospdo adoption namespace changes simpler (#856)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_preparing-controller-nodes-for-director-operator-adoption.adoc

.Procedure

. Log in to the {OpenShiftShort} environment where {OpenStackShort} OSPdO is deployed and change to the project that hosts your {OpenStackShort} deployment:
[NOTE]

> In Director Operator adoption you must define which namespace is being used for resources manipulation.
> Since with Director Operator the adopted namespace would be "openstack", the adopting namespace ,for rhoso , would be "rhoso".

Reviewer: ```suggestion
In director Operator adoption, you must define which namespace is being used for resources manipulation.
For director Operator adoption, the adopted namespace is `openstack`. If you are adopting a RHOSO environment, the adopted namespace is `rhoso`.
```
I'm not sure if we need the last sentence about RHOSO, since this procedure only applies to director Operator.
